earthsim/grabcut.py

```diff
@@ -181,8 +181,8 @@ class GrabCutPanel(param.Parameterized):
 
     def __init__(self, image, fg_data=[], bg_data=[], **params):
         super(GrabCutPanel, self).__init__(image=image, **params)
-        self._bg_data = bg_data
-        self._fg_data = fg_data
+        self._bg_data = gv.project(self.path_type(bg_data, crs=self.crs), projection=ccrs.GOOGLE_MERCATOR)
```

Comment:
@philippjfr, @kcpevey Quest now returns images from the WMTS service in the Google Mercator projection. This caused some issues with this file. Kim helped me figure out what was going on yesterday, and these changes are what made sense to me to get it working, but let me know if you have any suggestions.

Reply:
This should be fine, but doesn't necessarily have to happen ahead of time. Actually I think the old logic probably would have been fine too. Previously I think it would have gone:

Declared CRS -> Image projection

while now it goes:

Declared CRS -> Google Mercator -> Image projection

but since, if I understand you correctly the image projection is Google Mercator that last transform is a no-op.